imagersite/photo/models.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class Photos(models.Model):
+    image = models.ImageField(upload_to='photo_files/%y-%m-%d')
+    user = models.ForeignKey(User, null=False)
+    title = models.CharField(max_length=256)
+    description = models.TextField()
+    date_uploaded = models.DateField(auto_now_add=True)
+    date_modified = models.DateField(auto_now=True)
+    date_published = models.DateField()
+
+    def __str__(self):
+        return self.title
+
+
+@python_2_unicode_compatible
+class Album(models.Model):
+    user = models.ForeignKey(User, null=False)
+    photos = models.ForeignKey(Photos, related_name='albums')
+    cover = models.ForeignKey(Photos, related_name='cover_for')
```

Comment:
Don't leave blank lines between field attributes of your model.  Group them all together for clarity of construction.